types/mclimate/vicki/uplink.js

```diff
@@ -290,6 +291,17 @@ function consume(event) {
   delete data.lowMotorConsumption;
   delete data.brokenSensor;
 
+  // Add raw metadata for mclimate integration. Sent to mclimate broker.
+  raw.deviceId = event.device.deviceId;
+  raw.payloadHex = event.data.payloadHex;
+  raw.timestamp = event.uplinkMetrics.timestamp;
+  raw.port = event.uplinkMetrics.port;
+  raw.frameCountUp = event.uplinkMetrics.frameCountUp;
+  raw.rssi = event.uplinkMetrics.rssi;
+  raw.snr = event.uplinkMetrics.snr;
+  raw.spreadingFactor = event.uplinkMetrics.sf;
+
   emit("sample", { data, topic: "default" });
   emit("sample", { data: lifecycle, topic: "lifecycle" });
+  emit("sample", { data: raw, topic: "mclimate" });
```

Comment:
is there a specific need to store it in akenza that way? if not, I would directly use payload templating to access these values or at least remove the values that can be accessed with payload templating 

Also I would likely rename the topic to raw_payload